authorization/http.go

```diff
@@ -42,7 +42,9 @@ func WithAuthorizers(authorizers map[string]rbac.Authorizer, permission rbac.Per
 				return
 			}
 
-			if statusCode, ok := a.Authorize(subject, groups, permission, resource, tenant, token); !ok {
+			tenantID, _ := authentication.GetTenantID(r.Context())
```

Comment:
In all of the context value getters above we handle the `!ok` case and return an error to the user. Can we do the same here?